src/Run/Setup.cs

```diff
@@ -484,11 +484,11 @@ private string GetProject(Command commandToExecute, List<string> parametersSelec
         public string FormatSetting(string option, string value, string type, string toolName)
         {
             string commandOption = null;
-            if (type.Equals("passThrough"))
+            if (type != null && type.Equals("passThrough"))
```

Comment:
I've already tried the null-coalescing operator here and got a strange error, so did this the old-fashioned way.